frontend/src/FullPageRoutes/VideoViewPage.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react"
+import useFocusedUser from "../utils/context"
+import { request } from "../utils/fetch"
+import Video from "../components/video/Video"
+import { Dialog } from "@material-ui/core"
+
+type VideoType = {
+  video_file: string
+  id: number
+  receiver: number
+}
+
+export default function VideoViewPage(): JSX.Element {
+  // In this case, focusedUser will be the one whose video I am viewing
+  const [focusedUser] = useFocusedUser()
+  const [video, setVideo] = React.useState<VideoType>({
+    id: 0,
+    receiver: 0,
+    video_file: "",
+  })
+
```

Comment:
Can you explain to me why you don't just set the values in the type VideoType?

Reply:
Because `VideoType` is a type declaration, it can't hold values itself. But the variable `video`, which is of type `VideoType`, holds the values